delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/indexing/IndexingStreamCoordinator.scala

```diff
@@ -0,0 +1,49 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.indexing
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ActorRef, ActorSystem, SupervisorStrategy}
+import akka.cluster.typed.{ClusterSingleton, SingletonActor}
+import ch.epfl.bluebrain.nexus.delta.kernel.RetryStrategy
+import ch.epfl.bluebrain.nexus.delta.sdk.indexing.IndexingStreamCoordinatorBehavior.{IndexingStreamCoordinatorCommand, RestartIndexing, StartIndexing, StopIndexing}
+import ch.epfl.bluebrain.nexus.sourcing.projections.ProjectionId.ViewProjectionId
+import ch.epfl.bluebrain.nexus.sourcing.projections.{Projection, ProjectionProgress}
+import fs2.Stream
+import monix.bio.Task
+import monix.execution.Scheduler
+
+class IndexingStreamCoordinator[V, P] private (
+    ref: ActorRef[IndexingStreamCoordinatorCommand[V]],
+    projection: Projection[P],
+    idF: V => ViewProjectionId
```

Comment:
We can use `Mapper[V, ViewProjectionId]`. And I would put that as an implicit value